dali/kernels/signal/fft/cufft_helper.h

```diff
@@ -0,0 +1,114 @@
+// Copyright (c) 2020, NVIDIA CORPORATION. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef DALI_KERNELS_SIGNAL_FFT_CUFFT_HELPER_H_
+#define DALI_KERNELS_SIGNAL_FFT_CUFFT_HELPER_H_
+
+#include <cufft.h>
+#include <utility>
+#include <string>
+#include "dali/core/cuda_error.h"
+#include "dali/core/unique_handle.h"
+#include "dali/core/format.h"
+
+namespace dali {
+
+class CUFFTError : public std::runtime_error {
+ public:
+  explicit CUFFTError(cufftResult result, const char *details = nullptr)
+  : std::runtime_error(Message(result, details))
+  , result_(result) {}
```

Comment:
The initializer list is not in tact with [code style](https://google.github.io/styleguide/cppguide.html#Constructor_Initializer_Lists)

Reply:
Just. Stop.

Reply:
Thy why shall we have the code style in the first place if we don't comply with it?

Reply:
@mzient - if you think that this rule is harmful for the project lest us all agree that we should not apply it.